src/main/resources/db/migration/V27__INSERT_BOOK_AWARDS.sql

```diff
@@ -0,0 +1,133 @@
+BEGIN TRANSACTION;
+do
+$$
+    declare
+        bloomsbury_publishing  int;
+        harper_collins         int;
+
+        fantasy                int;
+        historical_fiction     int;
+
+        english                int;
+
+        hardcover              int;
+
+        clarke                 int;
+        mantel                 int;
+
+        womens_prize           int;
+        booker_prize           int;
+
+    begin
+
+        SELECT id
+        INTO english
+        FROM lang
+        WHERE name = 'English';
+
+        SELECT id
+        INTO fantasy
+        FROM genre
+        WHERE name = 'Fantasy';
+
+        SELECT id
+        INTO historical_fiction
+        FROM genre
+        WHERE name = 'Historical fiction';
+
+        SELECT id
+        INTO bloomsbury_publishing
+        FROM publisher
+        WHERE name = 'Bloomsbury Publishing';
+
+        SELECT id
+        INTO harper_collins
+        FROM publisher
+        WHERE name = 'Harper Collins';
+
+        SELECT id
+        INTO hardcover
+        FROM publishing_format
+        WHERE format_name = 'Hardcover';
+
+        INSERT INTO book (title,
+                          isbn13,
+                          genre_id,
+                          year_of_publication,
+                          blurb,
+                          publisher,
+                          isbn10,
+                          lang_id,
+                          publishing_format_id)
+        VALUES ('Piranesi',
+                '9781526622426',
+                fantasy,
+                2020,
+                'Piranesi''s house is no ordinary building: its rooms are infinite, its corridors endless, its walls are lined ' ||
```

Comment:
Where did you get the blurbs for this book and Wolf Hall?

Reply:
I got the blurbs from Goodreads:
[Piranesi](https://www.goodreads.com/book/show/50202953-piranesi)
[Wolf Hall](https://www.goodreads.com/book/show/6101138-wolf-hall)

Reply:
Ah, could you get the blurbs instead from the publisher's or author's website and also send a link here to the sources? I don't think we're able to use the content from Goodreads (e.g. if it was sourced from Amazon or some proprietary source)

Reply:
Adding links to the blurbs posted by the publishers:
[Bloomsbury - Piranesi](https://www.bloomsbury.com/us/piranesi-9781635575637/)
[4th Estate (HarperCollins) - Wolf Hall](https://www.4thestate.co.uk/book/wolf-hall-the-wolf-hall-trilogy-9780008366759/)